layouts/page/cudf-polars.html

```diff
@@ -0,0 +1,307 @@
+            {{ with .Resources.GetMatch "polars-gpu-cpu.png" }}
+            <img src="{{ .RelPermalink }}">
+            {{ end }}
+            <br>
+            <p class="fs-7 fw-lighter text-center">PDS-H is derived from TPC-H but these results are not comparable to TPC-H results. <a href="#" target="_blank">Learn More <i class="fa-solid fa-arrow-up-right"></i> </a></p>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+
+<!-- Get Started -->
+ <div id="get-started"></div>
+<section class="row section padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="mb-4"> How It Works</h1>
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Default GPU </h2>
+          <p>First, install the cuDF backend for Polars with a feature flag in your standard Python pip install command:</p>
+          <code class="code-sample mt-3">
+            pip install polars[gpu] —extra-index-url=https://pypi.nvidia.com
+          </code>
+         
+          <br>
+
+          <p>To use the default GPU backend configuration, simply specify the backend as GPU to materialize the Polars' LazyFrame into a DataFrame using <code>.collect()</code> API:</p>
+          <code class="code-sample mt-3">
+            import polars as pl
+            
+            <br><br>
+
+            ldf = pl.LazyFrame({
+                "a": [1.242, 1.535],
+            })
+
+            <br><br>
+
+            print(ldf.select(
+                  pl.col("a").round(1)
+                  ).collect(engine="gpu"))
+          </code>
+        </div>
+        <div class="col-md-6 py-3">         
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Custom GPU </h2>
+          <p>First, install the cuDF backend for Polars with a feature flag in your standard Python pip install command:</p>
+          <code class="code-sample mt-3">
+            pip install polars[gpu] —extra-index-url=https://pypi.nvidia.com
+          </code>
+          
+          <br>
+
+          <p>For finer control on the backend, you can pass a <code>GPUEngine</code> object with additional configurations like device details and verbosity to the <code>engine=</code> parameter:</p>
+          <code class="code-sample mt-3">
+            import polars as pl
+           
+            <br><br>
+           
+            ldf = pl.LazyFrame({
+                "a": [1.242, 1.535],
+            })
+           
+            <br><br>
+
+            gpu_engine = pl.GPUEngine(
+                device=0, # This is the default
+                raise_on_fail=True, # Fail loudly if can't execute on the GPU
+                )
+
+            <br><br>
+            
+            print(ldf.select(pl.col("a").round(1)
+            ).collect(engine=gpu_engine))
+            
+          </code>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+
+<!-- FAQ -->
```

Comment:
Alan - Can we add FAQs in a sequential format with an accordian clickable format? This one seems a little tough to read right now. 

<!-- FAQ Section with Accordion -->
<section class="row section bg-rapids-gray padding-top-slant padding-bottom-slant">
  <div class="col">
    <div class="container">
      <h1 class="mb-4">Frequently Asked Questions</h1>
      
      <!-- Accordion Container -->
      <div class="accordion" id="faqAccordion">

        <!-- Accordion Item 1 -->
        <div class="accordion-item">
          <h2 class="accordion-header" id="headingOne">
            <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
              Do I have to install any RAPIDS specific package for the GPU backend to work?
            </button>
          </h2>
          <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne" data-bs-parent="#faqAccordion">
            <div class="accordion-body">
              No, you just need to install Polars with a feature flag in your standard Python pip install command: 
              <br><code>pip install polars[gpu]--extra-index-url=https://pypi.nvidia.com</code>
            </div>
          </div>
        </div>

        <!-- Accordion Item 2 -->
        <div class="accordion-item">
          <h2 class="accordion-header" id="headingTwo">
            <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
              Can I use the cuDF backend for Polars on my local Windows machine?
            </button>
          </h2>
          <div id="collapseTwo" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#faqAccordion">
            <div class="accordion-body">
              Yes, you can use the cuDF backend for Polars on a Windows machine by installing WSL2. Note: You would need WSL2 on a Windows 11 or above OS. For download and troubleshooting instructions, please visit <a href="#" target="_blank">RAPIDS installation</a> page.
            </div>
          </div>
        </div>